<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransparencySlider</class>
 <widget class="QWidget" name="TransparencySlider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>171</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>351</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>281</width>
       <height>74</height>
      </size>
     </property>
     <property name="title">
      <string>Маскирование</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Черный</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="applianceBox">
       <property name="toolTip">
        <string>Список слоев</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lockButton">
       <property name="maximumSize">
        <size>
         <width>41</width>
         <height>34</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Заблокировать изменения</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
